### PR TITLE
fix(Core/Spells): Correct ID of Warlock spellscript

### PR DESCRIPTION
**Pull request by poszer**
Use correct spell ID for Shadow Ward (Warlock spell)

##### CHANGES PROPOSED:

-  Set correct ID of warlock spell Shadow Ward.  It was set to -7235(Shadow Protection) which is wrong
https://wotlk.evowow.com/?spell=7235
-  Correct is to use -6229 (Shadow Ward)
https://wotlk.evowow.com/?spell=6229
- Also updated DB `world` -> `spell_script_names` to match these changes

###### ISSUES ADDRESSED:

Closes 

##### TESTS PERFORMED:

Yes, everything fine.

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1560493947655900000.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1560493947655900000');
+
+UPDATE `spell_script_names` SET `spell_id`=-6229 WHERE `ScriptName`='spell_warl_shadow_ward';
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1560493947655900000.sql]:
```suggestion
UPDATE `spell_script_names` SET `spell_id`=6229 WHERE `ScriptName`='spell_warl_shadow_ward';
```

Comment (lineagedr) [reply data/sql/updates/pending_db_world/rev_1560493947655900000.sql]: @BarbzYHOOL We need that minus to check for all ranks of the spell. Without it the spell script will only work for rank 1

Comment (poszer) [reply data/sql/updates/pending_db_world/rev_1560493947655900000.sql]: @BarbzYHOOL no, it must be with - to check ranks. Just like @lineagedr explained

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_world/rev_1560493947655900000.sql]: ok thank you, i suspected it was not a mistake but i wasn't sure :P

**Comment by BarbzYHOOL** [comment]
I'm wondering why such a simple spell requires its own script. I'm sure there are values for everything in the DBC (and so the server knows how to interpret them, normally)

**Comment by lineagedr** [comment]
> I'm wondering why such a simple spell requires its own script. I'm sure there are values for everything in the DBC for that spell, even the shadow resist (and so the server knows how to interpret them, normally)

I believe it needs it's own spell script because of:

sw

**Pull request merged**